lib/sequent/generator/template_project/db/migrate/1_add_event_store.rb

```diff
@@ -0,0 +1,54 @@
+class AddEventStore < ActiveRecord::Migration[4.2]
```

Comment:
Why is this `4.2`? Also this PR don't test against `4.2` anymore, see travis.yml.